src/components/FormRegister.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <form class="form-wrapper" @submit.prevent="onSubmit">
+    <BaseInput
+      type="email"
+      label="email"
+      placeholder="Enter your email address"
+      v-model="email"
+      @blur="isEmailTouched = true"
+    />
+    <div>{{ emailError }}</div>
+    <PasswordInput
+      label="password"
+      v-model="password"
+      @blur="isPasswordTouched = true"
+    />
+    <div>{{ passwordError }}</div>
+    <div class="form-wrapper__hints">
+        <p>At least 8 characters</p>
+        <p>At least one letter</p>
+        <p>At least one digit</p>
+    </div>
+    <div class="form-wrapper__action">
+      <BaseButton caption="Log in instead" buttonType="secondary"/>
+      <BaseButton caption="Next step"/>
```

Comment:
which button is `type="submit"`?

---
`type="submit"` is the default if the attribute `type` is not specified for buttons associated with a `<form>`, or if the attribute is an empty or invalid value.

to prevent it you need to specify other type for a button e.g. `type="button"`

more: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type